typing_extensions/src/typing_extensions.py

```diff
@@ -1231,6 +1231,27 @@ class Film(TypedDict):
         """
         return isinstance(tp, tuple(_TYPEDDICT_TYPES))
 
+
+if hasattr(typing, "assert_type"):
+    assert_type = typing.assert_type
+
+else:
+    def assert_type(__val, __typ):
+        """Assert (to the type checker) that the value is of the given type.
```

Comment:
More a critique of the original PR, but not sure that "Assert (to the type checker) that *val* has an inferred type of *typ*" is the least confusing phrasing.

You're not telling the type checker a fact, you're asking them to confirm it. Maybe "Ask a static type checker to confirm that the value is of the given type".

Or at least, we should do enough to do more to distinguish it from `assert isinstance(val, typ)`, which is what I would describe as "asserting something to the type checker"

Reply:
Could you bring this up on cpython instead? The docstring here is just a copy of the CPython one. We can sync it back later if we want.

Reply:
Sure. Should I open a PR directly against the original BPO or would you prefer I file its own BPO

Reply:
We can keep the same one